bin/PCA.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from sklearn.decomposition import PCA
+import plotly.express as px
+import argparse
+import numpy as np
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-I','--input',required=True, help='path of input file (csv)')
+parser.add_argument('-O','--output',required=True, help='path of output (plots)')
+parser.add_argument('-S','--seperator',default=',',help='column separator of your input csv, default = ,')
+parser.add_argument('-C','--color',default='Plotly', help='color pallette (plotly Discrete Colors, https://plotly.com/python/discrete-color/)')
+parser.add_argument('-CC','--colourCol',required=True, help='column name which indicates the column which will be used to colercode the PCA-plot.\nif a comma separated list is pased the programm will create a plot for each element')
+parser.add_argument('-D','--dimensions',default=2,help='indicates wheather a 3 or 3 dimensional plot is created (default = 2)')
+parser.add_argument('-A','--annotationColumns',default=None, help='comma separated list fo column names with annotations, not variabl data (default = None')
+parser.add_argument('-N','--plotName', default='PCA',help='Plot title (default = PCA')
+parser.add_argument('-SN','--sampleName',default=None, help='Column name with names of samples wich are used as lables for the dots of the PCA-plot (default = None')
+parser.add_argument('-F','--figureType',default='png',help='File type of the plots in the output (Default = png)')
+
+args = parser.parse_args()
+data_path = args.input
+output_path = args.output
+sep = args.seperator
+color = args.color
+colC = args.colourCol
+dim = args.dimensions
+Annotations = args.annotationColumns
```

Comment:
Sorry for being nitpicky, but per convention, in Python, all variables are lower case, while only classes start with a lower case letter.

Reply:
Corrected